packages/hardhat-core/test/internal/hardhat-network/stack-traces/compilers-list.ts

```diff
@@ -209,4 +211,26 @@ export const solidityCompilers: SolidityCompiler[] = [
     },
     latestSolcVersion: true,
   },
+  {
+    solidityVersion: "0.8.21",
+    compilerPath: "soljson-v0.8.21+commit.d9974bed.js",
+    latestSolcVersion: true,
+  },
+  {
+    solidityVersion: "0.8.21",
+    compilerPath: "soljson-v0.8.21+commit.d9974bed.js",
+    optimizer: {
+      runs: 200,
+      viaIR: true,
+    },
+    latestSolcVersion: true,
+  },
 ];
+
+export const getLatestSupportedVersion = () =>
+  solidityCompilers.map((sc) => sc.solidityVersion).sort(semver.compare)[
+    solidityCompilers.length - 1
+  ];
+
+export const increasePatch = (version: string): string =>
```

Comment:
Personal preference: I would either inline this where it's used (because `semver.inc(version, "patch")` it's pretty self-explanatory IMO) or I would generate and export the `nextVersion` and `nextNextVersion` variables here. This feels like a weird middle ground where we are exporting a generic unrelated functionality.

Reply:
Yeah, this started with a self-made function to increase the patch version and then I remembered semver already had that built-in. I'll import it directly on the config files.